everyvoice/_version.py

```diff
@@ -1,5 +1,5 @@
-# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)
-# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)
-# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)
+# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)  # noqa: E501
+# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)  # noqa: E501
+# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)  # noqa: E501
```

Comment:
With the global E501 exclusion in `.flake8` these noqa comments are not necessary.